config/crd/full/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
We also need to update the crd helm chart, probably need to a make chart for minimal crd

Reply:
Added kserve-crd-minimal chart